core/templates/domain/topic/TopicObjectFactory.ts

```diff
@@ -613,7 +614,7 @@ export class TopicObjectFactory {
   // the actual topic is fetched from the backend.
   createInterstitialTopic(): Topic {
     return new Topic(
-      null, 'Topic name loading', 'Abbrev. name loading',
+      '', 'Topic name loading', 'Abbrev. name loading',
```

Comment:
Why this change? `''` is not good replacement for `null`

Reply:
Done now